guides/source/getting_started.md

```diff
@@ -186,6 +186,30 @@ of the files and folders that Rails created by default:
 |.gitignore|This file tells git which files (or patterns) it should ignore. See [GitHub - Ignoring files](https://help.github.com/articles/ignoring-files) for more info about ignoring files.
 |.ruby-version|This file contains the default Ruby version.|
 
+Dockerfile
+-----------
+
+Rails will generate all the Dockerfiles you need to deploy your application using
+Kamal, or any other Docker-based deployment setup, when you run rails new. These 
+Dockerfiles are tuned for production use with proper caching layers, multi-stage building
+to minimize image sizes, and all the dependencies needed whether you use a JavaScript 
+build environment or not. You can use Docker to run your application by running the 
+following commands:
+
+```bash
+$ cat config/master.key
+very-secret-master-key
+$  docker run -p 3000:3000 -e RAILS_MASTER_KEY="very-secret-master-key" -e DATABASE_URL=postgresql://xxx:@host.docker.internal/rails_50402_development rails-50402
+```
```

Comment:
I guess we need to build docker image first. And later while running the docker run instead of doing cat as a separate command. We can merge two command. I have also dropped Postgres from environment variables.

```bash
docker build -t rails-prod-build .
docker run -p 3000:3000 -e RAILS_MASTER_KEY="$(cat config/master.key)" rails-prod-build
```